### PR TITLE
TICK-278 documentation improvement

### PR DESCRIPTION
**Pull request by Mbompr**

### CODE REVIEW

File: DOC/modules/optim.rst [hunk 1]
```diff
@@ -0,0 +1,517 @@
...
+Solver                                                   Class
+=======================================================  ========================================
+Proximal gradient descent                                :class:`GD <tick.optim.solver.GD>`
+Accelerated proximal gradient descent                    :class:`AGD <tick.optim.solver.AGD>`
+Broyden, Fletcher, Goldfarb, and Shannon (quasi-newton)  :class:`BFGS <tick.optim.solver.BFGS>`
+Self-Concordant Proximal Gradient Descent                :class:`SCPG <tick.optim.solver.SCPG>`
+Stochastic Gradient Descent                              :class:`SGD <tick.optim.solver.SGD>`
+Stochastic Variance Reduced Descent                      :class:`SVRG <tick.optim.solver.SVRG>`
+Stochastic Dual Coordinate Ascent                        :class:`SDCA <tick.optim.solver.SDCA>`
+=======================================================  ========================================
```
Comment (stephanegaiffas) [inline DOC/modules/optim.rst]: We forgot AdaGrad !!!

File: DOC/conf.py [hunk 2]
```diff
@@ -71,11 +82,12 @@
 
 # General information about the project.
 project = 'tick'
-copyright = '2015, Emmanuel Bacry, Martin Bompaire, Stephane Gaiffas, ' \
+copyright = 'Emmanuel Bacry, Martin Bompaire, Stephane Gaiffas, ' \
```
Comment (Mbompr) [inline DOC/conf.py]: http://ben.balter.com/2015/06/03/copyright-notices-for-websites-and-open-source-projects/#the-copyright-holder

It seems that we are supposed to set it to the copyright holder. Is it Ecole polytechnique in our case?

**Comment by Mbompr** [comment]
We still need to make up our mind on copyright. Maybe should we create a task including this and a decision about headers? 
http://jmlr.org/mloss/mloss-info.html

> Ensure the licensing terms of all included components comply with the open source definition and are clearly stated in the source package and mentioned in each source file

**Comment by Mbompr** [comment]
@stephanegaiffas Don't know if you have received a notification

File: DOC/conf.py (hunk 2 above)
Comment (stephanegaiffas) [inline DOC/conf.py]: I don't know... let's do it in another task

**Pull request merged**